workers/ohsome_quality_analyst/oqt.py

```diff
@@ -67,8 +71,9 @@ async def _(
         logging.info("Input feature identifier: " + id_)
         if size_restriction:
             await check_area_size(feature.geometry)
-        indicator = await create_indicator(parameters.copy(update={"bpolys": feature}))
-        features.append(indicator.as_feature(flatten=True))
+        tasks.append(create_indicator(parameters.copy(update={"bpolys": feature})))
+    indicators = await asyncio.gather(*(sem_task(task) for task in tasks))
+    features = [i.as_feature(flatten=True) for i in indicators]
```

Comment:
Do you understand the [documentation of asyncio.gather(…)](https://docs.python.org/3/library/asyncio-task.html#asyncio.gather) that the order of the results is related to the order of the inputs/tasks? Otherwise we would need tests. I don't think we need one.

Reply:
Yes, I do interpret it that way!

 > The order of result values corresponds to the order of awaitables in *aws*.

(*aws* is the sequence of awaitables)